From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sat, 19 Mar 2016 19:02:15 +0100
Subject: [PATCH] FIXME: Disable amd64 Newton on Ubuntu for now.

Until we fix all cohabitation of amd64 and arm64, disable Openstack
releases <Newton on Ubuntu 16.04> and <Newton on Ubuntu+UCA 16.04>
(both x86_64) in openstack.yaml fixture.

TODO: Propose renaming above releases to include arch in name.
Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 nailgun/nailgun/fixtures/openstack.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/nailgun/nailgun/fixtures/openstack.yaml b/nailgun/nailgun/fixtures/openstack.yaml
--- a/nailgun/nailgun/fixtures/openstack.yaml
+++ b/nailgun/nailgun/fixtures/openstack.yaml
@@ -1980,6 +1980,7 @@
   extend: *base_release
   fields:
     name: "Newton on Ubuntu 16.04"
+    state: "unavailable"
     version: "newton-10.0"
     arch: "amd64"
     operating_system: "Ubuntu"
@@ -2221,5 +2222,6 @@
   fields:
     name: "Newton on Ubuntu+UCA 16.04"
+    state: "unavailable"
     version: "newton-10.0"
     arch: "amd64"
     description: "This option will install the OpenStack Newton packages using Ubuntu as a base operating system, including Ubuntu Cloud Archive OpenStack packages. With high availability features built in, you are getting a robust, enterprise-grade OpenStack deployment."
